statsmodels/tsa/automatic/forecast.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+import pandas as pd
+from scipy.stats import norm
+from statsmodels.tools.decorators import cache_readonly
+
+
+class Forecast:
+    def __init__(self, endog, model, test_sample=0.2, **spec):
+        if type(test_sample) == str:
```

Comment:
This is probably okay for now, but it is not robust to various types of endog, etc. Can you add a TODO comment in the code as a reminder? Something like:

```python
# TODO: make date selection of test sample more robust
```